configure.ac

```diff
@@ -1378,6 +1383,11 @@ if test "$PHP_MEMORY_SANITIZER" = "yes"; then
   CXXFLAGS="$CXXFLAGS -fsanitize=memory -fsanitize-memory-track-origins"
 fi
 
+if test "$PHP_ADDRESS_SANITIZER" = "yes"; then
+  CFLAGS="$CFLAGS -fsanitize=address -fno-omit-frame-pointer"
```

Comment:
`-fno-omit-frame-pointer` needed? I never had to use it for asan builds.

Should be include `-DZEND_TRACK_ARENA_ALLOC`?

It would be good to update the `azure-pipelines.yml` to make use of the option, now that it exists. Currently it's explicitly passing CFLAGS.

Reply:
Whether it looks better now?

Reply:
`-fno-omit-frame-pointer` can help us to get nicer stack traces in error messages.
Although it is unnecessary when we use `--enable-debug(-O0)` but I think it is always better to do that.

Reply:
Hm okay, I guess it doesn't hurt :)